sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationOperation.cs

```diff
@@ -445,22 +474,7 @@ public override AsyncPageable<DocumentStatusDetail> GetValuesAsync()
         {
             ValidateOperationStatus();
 
-            async Task<Page<DocumentStatusDetail>> NextPageFunc(string nextLink, int? pageSizeHint)
-            {
-                // TODO: diagnostics scope?
-                try
-                {
-                    Response<DocumentStatusResponse> response = await _serviceClient.GetOperationDocumentsStatusNextPageAsync(nextLink, new Guid(Id)).ConfigureAwait(false);
-
-                    return Page.FromValues(response.Value.Value, response.Value.NextLink, response.GetRawResponse());
-                }
-                catch (Exception)
-                {
-                    throw;
-                }
-            }
-
-            return PageableHelpers.CreateAsyncEnumerable(_ => Task.FromResult(_firstPage), NextPageFunc);
+            return GetAllDocumentsStatusAsync();
```

Comment:
does this needs to be return await GetAllDocumentsStatusAsync() ?

Reply:
@maririos No the function is not actually async it just returns an AsyncPageable

Reply:
duh. Right!